### PR TITLE
IBM watson translator integration

### PR DESCRIPTION
**Pull request by lpawelcz**

### CODE REVIEW

**Review by wojtasiq — APPROVED** [review]

**Review by jakbal — CHANGES_REQUESTED** [review]
1) Change class name to TextTranslator instead TextTransl (where 'l' looks like '1')
2) Try upgrading flutter_ibm_watson to 0.0.4 as it's the newest one.

File: pubspec.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ dependencies:
   camera: ^0.5.6+1
   firebase_core: 0.5.0
   image_picker: ^0.6.7+17
-  firebase_mlkit_language: ^1.1.3
+  flutter_ibm_watson: ^0.0.1
```
Comment (lpawelcz) [inline pubspec.yaml]: NOTE: Version change requested by @belikePac

File: lib/translation/text-transl.dart [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import 'dart:convert' show json;
+import 'package:flutter/services.dart' show rootBundle;
+import 'package:firebase_ml_vision/firebase_ml_vision.dart';
+import 'package:flutter_ibm_watson/language_translator/LanguageTranslator.dart';
+import 'package:flutter_ibm_watson/utils/IamOptions.dart';
+
+class TextTransl {
```
Comment (lpawelcz) [inline lib/translation/text-transl.dart]: NOTE: Change to `TextTranslator` as @belikePac  suggested

**Review by wojtasiq — APPROVED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (lpawelcz) [reply pubspec.yaml]: changed to 0.04v

File: lib/translation/text-transl.dart (hunk 2 above)
Comment (lpawelcz) [reply lib/translation/text-transl.dart]: renamed

**Comment by lpawelcz** [comment]
Applied changes requested in @belikePac review

**Review by jakbal — APPROVED** [review]
Very good work!

**Pull request merged**